### PR TITLE
config: add myself to the developer list

### PR DESCRIPTION
**Pull request by pbludov**
minor change to `pom.xml`
from https://github.com/checkstyle/checkstyle/pull/7265#issuecomment-554670610

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -45,6 +45,13 @@
         <role>project admin, developer</role>
       </roles>
     </developer>
+    <developer>
+      <id>pbludov</id>
+      <name>Pavel Bludov</name>
+      <roles>
+        <role>developer</role>
```
Comment (rnveach) [inline pom.xml]: @romani Please confirm...

Comment (romani) [reply pom.xml]: confirming with pleasure....

**Comment by rnveach** [comment]
@pbludov spellchecker failed

**Comment by pbludov** [comment]
> @pbludov spellchecker failed

done. I didn't know that `releasenotes.xml` is excluded from spell checking. My name is definitely there.

File: pom.xml (hunk 1 above)
Comment (strkkk) [reply pom.xml]: @pbludov congrats :)

**Comment by romani** [comment]
@pbludov , please merge this PR yourself

**Comment by pbludov** [comment]
> @pbludov , please merge this PR yourself

Access has not yet materialized:

image

Maybe github need some time to sync.

**Comment by romani** [comment]
Recheck email for invite

**Comment by pbludov** [comment]
Still not allowed to merge:
image

**Comment by romani** [comment]
strange ... 
You do have Write permissions ..
image

I removed you from list of Collaborators.
please recheck one more time ....

**Comment by romani** [comment]
Ok, looks like attentive reading of message give me extra insight on where to change settings ....

I did update at https://github.com/checkstyle/checkstyle/settings/branch_protection_rules/6055665
image

**Comment by pbludov** [comment]
Done. Thanks!

**Pull request merged**